### PR TITLE
Deprecate cpu and memory exceeds alerts

### PR DESCRIPTION
**Pull request by avlitman**
Deprecate the KubeVirtComponentExceedsRequestedCPU and KubeVirtComponentExceedsRequestedMemory alerts, which the user can't fix.

jira-ticket: https://issues.redhat.com/browse/CNV-37900

**Special notes for your reviewer**:

- [CNV-38320](https://issues.redhat.com/browse/CNV-38320) will handle adding a metric to be able to track this issue without creating noise to the user.

```release-note
KubeVirtComponentExceedsRequestedCPU and KubeVirtComponentExceedsRequestedMemory alerts are deprecated
```

### CODE REVIEW

File: alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md
Comment (orenc1) [inline alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md:3]: if you want this to be splitted into two lines at the rendered markdown page, you should add two spaces at the end of the first line.
```suggestion
This alert has been deprecated; it does not indicate a genuine issue.  
```

Comment (orenc1) [inline alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md:3]: also, i'm not sure _genuine_ is the right terminology to describe that. Genuine means sincere, truthful, which is not so relevant in this context.
i would suggest "real" or "actual".

Comment (avlitman) [reply alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md:3]: are you sure? I tested it in both states (with and without the space and same results), see https://github.com/openshift/runbooks/blob/8434876c33a38463387320f3880336ffc4c30068/alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md

Comment (avlitman) [reply alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md]: fixed

**Review by orenc1 — APPROVED** [review]
/approve

**Comment by sradco** [comment]
@ctomasko please let me know who should review and update the run books at this point. Thanks.

**Comment by avlitman** [comment]
/ hold until doc team will approve this.

**Comment by ctomasko** [comment]
> @ctomasko please let me know who should review and update the run books at this point. Thanks.

@sradco @avlitman  I have 2 writers who are updating the runbooks for v4.16. Is this alert being deprecated for v4.16 only, or does this apply to previous versions? If it's 4.16, we have not sync'd the upstream/dowstream yet, see epic https://issues.redhat.com/browse/CNV-36859.  
If this pertains to previous or the current 4.15 version, I'll need to know that I can open a Jira for the writing team.

**Comment by sradco** [comment]
/hold
@ctomasko Should we merge updated to this repo or only create PRs and ask for your review?

File: alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md
Comment (ousleyp) [inline alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md:4]: Is this the established text for a deprecated alert? If you're open to changing it, I think we can make it clearer by using minimalism principles.

- I'd suggest using the active voice ("You can safely ignore or silence this alert") instead of the passive voice ("it may be safely ignored and silenced").

- I'd remove "it does not indicate an actual issue" because it adds some confusion, IMO, and "safely" implies that it's a non-issue. 

Suggestion:

```suggestion
This alert is deprecated. You can safely ignore or silence it.
```

**Review by ousleyp — COMMENTED** [review]
Hi @sradco, I'll be happy to provide doc review. Just one comment that applies to both files. Thanks!

**Comment by avlitman** [comment]
Hi @ousleyp will appreciate if you can review again. also should we merge updates to this repo or only create PRs and ask for your review?

**Pull request merged**
